go-controller/pkg/config/cni.go

```diff
@@ -11,14 +12,17 @@ import (
 	"github.com/containernetworking/cni/pkg/types"
 	"github.com/containernetworking/cni/pkg/version"
 
-	ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types"
+	ovncnitypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types"
+	ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 )
 
+var ErrorAttachDefNotOvnManaged = errors.New("net-attach-def not managed by OVN")
+
 // WriteCNIConfig writes a CNI JSON config file to directory given by global config
 // if the file doesn't already exist, or is different than the content that would
 // be written.
 func WriteCNIConfig() error {
-	netConf := &ovntypes.NetConf{
+	netConf := &ovncnitypes.NetConf{
```

Comment:
yes, I changed to old import name, because we imported _ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"_

Reply:
What I mean is that if we could use `ovntypes.NetConf` instead of `ovncnitypes.NetConf` anyway, because we don't want to write any of the new NetConf properties. This method is used to write a standard NetConf without any new attributes like topology, etc...

Reply:
you mean types.NetConf? But today's ovn cni already relies on the logFile related filed to setup logging.

Reply:
Sorry, I was wrong here. I though we were changing the NetConf type used but we are not.